### PR TITLE
Update example versions for latest release

### LINKED ISSUE #47: Make Docker (and other?) actions fork-safe
The docker build action always tries to push to GHCR, and thus will fail for forks as they don't have permission to push to GHCR. We should either figure out how to allow pushes from forks (maybe not the best idea from a sec perspective!) or skip the push action for forks.

### PR DESCRIPTION
**Pull request by jhkennedy**
Also recommend using GITHUB_TOKEN for `reusable-docker-ghcr.yml` which fixes #47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,8 +25,7 @@ on:
 
 jobs:
   call-bump-version-workflow:
-    uses: ASFHyP3/actions/.github/workflows/reusable-bump-version.yml@v0.7.0
-    with:
+    uses: ASFHyP3/actions/.github/workflows/reusable-bump-version.yml@v0.7.1
```
Comment (jhkennedy) [inline README.md:28]:
```suggestion
    uses: ASFHyP3/actions/.github/workflows/reusable-bump-version.yml@v0.7.1
    with:
```

File: README.md [hunk 2]
```diff
@@ -25,8 +25,7 @@ on:
 
 jobs:
   call-bump-version-workflow:
-    uses: ASFHyP3/actions/.github/workflows/reusable-bump-version.yml@v0.7.0
-    with:
```
[the remark this answers is not in the document]
Comment (jhkennedy) [reply README.md:29]: yes! Goog catch!

File: README.md [hunk 3]
```diff
@@ -128,20 +128,21 @@ on:
...
     needs: call-version-info-workflow
-    uses: ASFHyP3/actions/.github/workflows/reusable-docker-ghcr.yml@v0.7.0
+    uses: ASFHyP3/actions/.github/workflows/reusable-docker-ghcr.yml@v0.7.1
     with:
       version_tag: ${{ needs.call-version-info-workflow.outputs.version_tag }}
+      user: ${{ github.actor }}
       release_branch: main     # Optional; default shown
       develop_branch: develop  # Optional; default shown
-      user: tools-bot          # Optional; default shown
+
```
Comment (jhkennedy) [inline README.md:143]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -295,7 +295,7 @@ on:
   
 jobs:
   call-release-workflow:
-    uses: ASFHyP3/actions/.github/workflows/reusable-release-checklist-comment.yml@v0.7.0
+    uses: ASFHyP3/actions/.github/workflows/reusable-relese-checklist-comment.yml@v0.7.1
```
Comment (jhkennedy) [inline README.md:298]:
```suggestion
    uses: ASFHyP3/actions/.github/workflows/reusable-release-checklist-comment.yml@v0.7.1
```

**Review by jacquelynsmale — APPROVED** [review]

**Pull request merged**